### PR TITLE
Create NoNetID screen and functionality

### PR DESCRIPTION
**Pull request by MStraight**
Added a NoNetID screen that is accessible from the EntryScreen. Everything should be functional, although someone with an iPhone should test the button functionality for the phone call. Design can be improved upon, but seems like a decent start.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +21,10 @@
     "react-native-reanimated": "~1.4.0",
     "react-native-safe-area-context": "0.6.0",
     "react-native-screens": "2.0.0-alpha.12",
+    "react-native-vector-icons": "^6.6.0",
```
Comment (nathan815) [inline package.json:24]: Is react-native-vector-icons used anywhere?

File: src/screens/NoNetIDPopup.tsx [hunk 2]
```diff
@@ -1,17 +1,99 @@
...
+      justifyContent: 'flex-start',
+      paddingLeft: 10,
+  },
+  phoneButton: {
+      backgroundColor: colors.seeusYellow,
+  },
+  titleText: {
+        fontSize: 30,
+        fontWeight: 'bold',
+        color: colors.black,
```
Comment (nathan815) [inline src/screens/NoNetIDPopup.tsx:77]: `white` and `black` don't exist on the colors object (styles/colors.ts file). Could you add them to that object?

**Comment by nathan815** [comment]
Testing on an iPhone SE (granted it's a small screen, and old) and it seems each element is vertically cut off:



Might need to utilize some more flexbox properties to fix this. I can help out if needed since I have this iPhone SE.

File: package.json (hunk 1 above)
Comment (MStraight) [reply package.json:24]: Ah, I forgot that these were added to the package.json file due to some experimenting that I was doing. I can all of the changes to that file, as I don't think any of them are needed.

File: src/screens/NoNetIDPopup.tsx (hunk 2 above)
Comment (MStraight) [reply src/screens/NoNetIDPopup.tsx:77]: Yup, just added them. I was wondering why they didn't seem to change anything (I kind of assumed there would be default colors like white and black to use).

Comment (nathan815) [reply src/screens/NoNetIDPopup.tsx:77]: yeah haha it was just a colors object I created; I should have added some default colors :D

**Comment by nathan815** [comment]
Good to merge this?

**Pull request merged**